docs/docusaurus/docs/components/connect_to_data/link_lists/_connect_to_filesystem_data_fluently.md

```diff
@@ -1,7 +1,7 @@
 **Local Filesystems**
-- [How to quickly connect to a single file using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/how_to_quickly_connect_to_a_single_file_with_pandas)
-- [How to connect to one or more files using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/how_to_connect_to_one_or_more_files_using_pandas)
-- [How to connect to one or more files using Spark](/docs/guides/connecting_to_your_data/fluent/filesystem/how_to_connect_to_one_or_more_files_using_spark)
+- [How to quickly connect to a single file using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
+- [How to connect to one or more files using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
+- [How to connect to one or more files using Spark](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
```

Comment:
Can we update these links so that they point to the appropriate tab on the page?  Otherwise, it seems excessive to have three links to the same spot.

Reply:
@Rachel-Reverie I still need to determine how to link to a specific tab. When time allows I'll do some more digging to determine how to implement this functionality.